charts/reform-scan-blob-router/values.yaml

```diff
@@ -31,6 +31,7 @@ java:
     CHECK_NEW_ENVELOPES_CRON: "0 0 * * * *"
     TASK_SCAN_DELAY: "4000" # in millis
     SEND_DAILY_REPORT_CRON: "0 0 18 ? * MON-FRI"
+    SEND_NOTIFICATIONS_CRON: "0 0/10 * * * *"
```

Comment:
Do we want to override for aat and preview ?

Reply:
Yeah, will update the PR

Reply:
Updated preview and aat values.yaml files

Reply:
aat shouldn't be updated but since we already applied such practice for this service - let it be so